### DISCUSSION TITLE
New Slack Link

### DISCUSSION THREAD

**Discussion by HarryET** [ChilliCream/hotchocolate#4060] [General]
Hey! I have tried both the slack link on the website and the link in the pinned discussion and neither work. Is there an up to date link I just haven't seen?

**Comment by PascalSenn** [comment]
Hi @HarryET  
Sorry for the inconvenience :) 
We will fix this asap
Meanwhile, you can join with this link:
https://join.slack.com/t/hotchocolategraphql/shared_invite/zt-pqkgbelg-yGDDppjvLOOdaGkdnNwRNA

**Reply by HarryET** [reply]
No Problems & Thanks for the link :)

**Reply by michaelstaib** [reply]
We have fixed the url now again: http://slack.chillicream.com is the actual link.

**Comment by PascalSenn** [comment]
Resolved with #4061
